app/src/main/java/org/oppia/android/app/topic/revisioncard/RevisionCardFragmentPresenter.kt

```diff
@@ -36,13 +43,23 @@ class RevisionCardFragmentPresenter @Inject constructor(
     val view = binding.revisionCardExplanationText
     val viewModel = getReviewCardViewModel()
 
-    viewModel.setSubtopicIdAndBinding(topicId, subtopicId, view)
+    viewModel.setTopicAndSubtopicId(topicId, subtopicId)
     logRevisionCardEvent(topicId, subtopicId)
 
     binding.let {
       it.viewModel = viewModel
       it.lifecycleOwner = fragment
     }
+
+    viewModel.revisionCardLiveData.observe(
+      fragment,
+      Observer {
+        view.text = htmlParserFactory.create(
+          resourceBucketName, entityType, topicId, /* imageCenterAlign= */ true
```

Comment:
```suggestion
          resourceBucketName, entityType, topicId, imageCenterAlign = true
```

Since this is a Kotlin function, prefer to use a named argument to have compiler verification.

Reply:
Sure, will do

Reply:
Done